app/src/main/res/layout/activity_login.xml

```diff
@@ -23,7 +23,6 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_below="@+id/studyHopperLogo"
```

Comment:
Is the logo gone from this activity's layout?

Reply:
no. it's line 14

Reply:
no, it's in line 14